conf/example.config.js

```diff
@@ -40,16 +40,28 @@ module.exports = {
 				if (err) {
 					return next(err);
 				}
-				if (body.hasOwnProperty('success')) {
-					if (!body.success) {
-						return next(body);
-					}
-					if (body.hasOwnProperty('key') && body.key && body.hasOwnProperty(body.key)) {
-						return next(null, body[body.key]);
+
+				// check if body exists first
+				if (typeof body === 'object') {
+					if (body.hasOwnProperty('success')) {
+						if (!body.success) {
+							return next(body);
+						}
+						if (body.hasOwnProperty('key') && body.key && body.hasOwnProperty(body.key)) {
+							return next(null, body[body.key]);
+						}
 					}
+
+					return next(null, body.hasOwnProperty('result') ? body.result : body);
 				}
 
-				next(null, body.hasOwnProperty('result') ? body.result : body);
+				if (response.statusCode === 201 && response.headers) {
+					// Extract created object's id from location string
```

Comment:
Instead of saying what you are doing, better explain why you are doing it. Currently the available documentation for the swagger connector is close to nonexistent, so everything helps! Since this is very specific to responses from our Arrow APIs, I think we should point that out. Other APIs usually don't need this as they return the created object in the response body.